roles/ceph-common/defaults/main.yml

```diff
@@ -8,7 +8,7 @@ ceph:
   #  - sdd
   #bcache_ssd_device: sdg 
 
-  stable_release: hammer
+  stable_release: jewel
```

Comment:
New mirrored location of ceph packages: https://apt-mirror.openstack.blueboxgrid.com/packagecloud.io/blueboxcloud/ceph-bbg/